diff --git a/source/CMakeLists.txt b/source/CMakeLists.txt
index 8102847..34ff5f1 100644
--- a/source/CMakeLists.txt
+++ b/source/CMakeLists.txt
@@ -165,7 +165,7 @@ if(GCC)
     add_definitions(-D__STDC_LIMIT_MACROS=1)
     add_definitions(-std=gnu++98)
     if(ENABLE_PIC)
-         add_definitions(-fPIC)
+         add_definitions(-fPIC -DPIC)
     endif(ENABLE_PIC)
     if(NATIVE_BUILD)
         if(INTEL_CXX)
diff --git a/source/test/checkasm-a.asm b/source/test/checkasm-a.asm
index 7a00098..acb6a29 100644
--- a/source/test/checkasm-a.asm
+++ b/source/test/checkasm-a.asm
@@ -156,7 +156,7 @@ cglobal checkasm_call, 2,15,16,max_args*8+8
     mov  r9, rax
     mov r10, rdx
     lea  r0, [error_message]
-    call puts
+    call puts wrt ..plt
     mov  r1, [rsp+max_args*8]
     mov  dword [r1], 0
     mov  rdx, r10
