diff -up x265_1.8/source/test/CMakeLists.txt.shared x265_1.8/source/test/CMakeLists.txt
--- x265_1.8/source/test/CMakeLists.txt.shared	2015-09-22 10:10:07.000000000 +0200
+++ x265_1.8/source/test/CMakeLists.txt	2015-10-24 12:15:04.846236808 +0200
@@ -23,7 +23,7 @@ add_executable(TestBench ${YASM_SRC}
     mbdstharness.cpp mbdstharness.h
     ipfilterharness.cpp ipfilterharness.h
     intrapredharness.cpp intrapredharness.h)
-target_link_libraries(TestBench x265-static ${PLATFORM_LIBS})
+target_link_libraries(TestBench x265-shared ${PLATFORM_LIBS})
 if(LINKER_OPTIONS)
     if(EXTRA_LIB)
         list(APPEND LINKER_OPTIONS "-L..")
